tasklist/client/src/modules/notifications/styles.module.scss

```diff
@@ -6,11 +6,12 @@
  * except in compliance with the Camunda License 1.0.
  */
 
-import {Union} from 'ts-toolbelt';
-import {themes} from './themes';
+@use '@carbon/layout';
 
-type CustomTheme = Union.Merge<typeof themes.g10 | typeof themes.g100>;
-
-declare module 'styled-components' {
-  export interface DefaultTheme extends CustomTheme {}
+.container {
+  position: absolute;
+  top: layout.to-rem(56px);
+  z-index: 1000;
+  right: var(--cds-spacing-03);
+  overflow: hidden;
```

Comment:
```suggestion
```

this doesn't seem to exist in the previous version 🤔

Reply:
I forgot to remove it, I was trying to fix an issue but it already exists and it only happens in rare occasions. This change didn't fix it.
I'll wait until someone complains since it only happens if a bunch of notifications are created at once.